docs/building.md

```diff
@@ -9,9 +9,14 @@ To build MongoDB, you will need:
     * Visual Studio 2017 version 15.9 or newer (See Windows section below for details)
 * On Linux and macOS, the libcurl library and header is required. MacOS includes libcurl.
     * Fedora/RHEL - `dnf install libcurl-devel`
-    * Ubuntu/Debian - `apt-get install libcurl-dev`
+    * Ubuntu/Debian - `libcurl-dev` is provided by three packages. Install one of them:
+      * `libcurl4-openssl-dev`
+      * `libcurl4-nss-dev`
+      * `libcurl4-gnutls-dev`
 * Python 3.7.x and Pip modules:
   * See the section "Python Prerequisites" below.
+* About 600 GB of free disk space (for source code and to build all components).
```

Comment:
I would make this more clear that this is only for the all target. Something like:

```
* About 13 GB of free disk space for the core binaries (`mongod`, `mongos`, and `mongo`) and about 600 GB for the all target.
```

Reply:
That sounds perfect. If you make it a suggestion, it'd be great (and preserve authorship).